openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -7,9 +7,15 @@
 {% if submission %}
     <h2 class="title">
         <span class="label staff-info__student__report__summary" tabindex="-1">
-            {% blocktrans with learner=student_username %}
-                Viewing learner: {{ learner }}
-            {% endblocktrans %}
+            {% if not is_team_assignment %}
```

Comment:
Trivial nit(for future reference): I prefer to have positive if statements unless absolutely necessary (to save my brain extra processing power). I.e: 
```
if is_team_assignment:
...
else:
```

Reply:
That's fair. I did this was mostly for flow: given that an individual assignment is overwhelmingly the default case, I want that to show first.